java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java

```diff
@@ -2,9 +2,11 @@
 
 import apps.StartupActionsManager;
 import apps.startup.StartupModel;
+import jmri.ConfigureManager;
 import jmri.InstanceManager;
 import jmri.configurexml.AbstractXmlAdapter;
 import org.jdom2.Element;
+import org.python.jline.internal.Log;
```

Comment:
Now I see what was happening, but not the why yet.
It seems that sometimes when I was adding an error message while I had typed Log, it wasn't seeing it as the same Log that the rest of them were. So it wasn't using the usual logger but pasting it its own idea.